app/models/ingest_job.rb

```diff
@@ -377,12 +379,13 @@ def set_study_state_after_ingest
     when :ingest_cell_metadata
       study.set_cell_count
       set_study_default_options
-      launch_subsample_jobs
+      launch_subsample_jobs unless study_file.is_anndata?
```

Comment:
Nice catch - this is one of the larger issues we have to tackle with AnnData ingest as we can't currently subsample with what we have